dipy/reconst/tests/test_dti.py

```diff
@@ -79,6 +79,15 @@ def test_tensor_model():
             npt.assert_almost_equal(dtifit.fa[0,0,0], dtifit_to_relative.fa,
                                     decimal=3)
 
+    # Check that it works on signal that has already been normalized to S0:
```

Comment:
Well, it's one of your commits:

https://github.com/samuelstjean/dipy/commit/996833c5ba2a1be4bef26b39f35d4e0e36d0efcb

You can interactively rebase that commit out and force push it again, to
eliminate it.

On Thu, Feb 19, 2015 at 1:45 PM, Samuel St-Jean <notifications@github.com>
wrote:

> In dipy/reconst/tests/test_dti.py
> <https://github.com/arokem/dipy/pull/12#discussion_r25028899>:
>
> > @@ -79,6 +79,15 @@ def test_tensor_model():
> >              npt.assert_almost_equal(dtifit.fa[0,0,0], dtifit_to_relative.fa,
> >                                      decimal=3)
> >
> > +    # Check that it works on signal that has already been normalized to S0:
>
> Not sure wtf that is doing here... Is that supposed to happen?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arokem/dipy/pull/12/files#r25028899>.
>